package.json

```diff
@@ -110,6 +108,7 @@
     },
     "mac": {
       "provisioningProfile": "keys/mac_production.provisionprofile",
+      "notarize": { "teamId": "TEAM-ID-HERE" },
```

Comment:
non-blocking: Can this be moved to a configuration file or an envvar?

This way the builder always has to edit the package.json file to build, so having it as a separate config file may be the best, unless we can put the team-id here (don't know if it has security issues)

Reply:
I agree it's a nuisance to have this identifier set like this. However, it's hard to fix it in a simple way.

1) It's not possible to interpolate environment variables into the `package.json` file

2) This configuration objects feeds into the `update-builder` implementation of calling the notarization scripts.
If we wanted to call the notarization without this specific script we would need to implement the notarization ourselves. This is precisely what we removed on the deleted `scripts/notarize.js` file.

That way, I think this is a technical debt small enough to be solved later by a more elegant solution.